src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Confirm.tsx

```diff
@@ -150,31 +152,30 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
 
       const quote = await getThorchainSaversDepositQuote({ asset, amountCryptoBaseUnit })
 
-      const { expected_amount_out, slippage_bps } = quote
+      const { expected_amount_out: expectedAmountOutThorBaseUnit, slippage_bps } = quote
 
-      setDepositFeeCryptoBaseUnit(
-        toBaseUnit(
-          fromThorBaseUnit(amountCryptoThorBaseUnit.minus(expected_amount_out)),
-          asset.precision,
-        ),
+      // Total downside
+      const depositFeeCryptoPrecision = fromThorBaseUnit(
+        amountCryptoThorBaseUnit.minus(expectedAmountOutThorBaseUnit),
       )
-      const percentage = bnOrZero(slippage_bps).div(BASE_BPS_POINTS).times(100)
 
-      // total downside (slippage going into position) - 0.007 ETH for 5 ETH deposit
+      setDepositFeeCryptoBaseUnit(toBaseUnit(depositFeeCryptoPrecision, asset.precision))
+
+      const slippagePercentage = bnOrZero(slippage_bps).div(BASE_BPS_POINTS).times(100)
+
+      // slippage going into position - 0.007 ETH for 5 ETH deposit
```

Comment:
```suggestion
      // slippage going into position - e.g. 0.007 ETH for 5 ETH deposit
```

i yolo'd this comment originally and don't want it to be taking literally as a fixed rate